Logos/Darktrace.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 89.76 99.09">
+  <defs>
+    <style>.cls-1,.cls-4,.cls-5{opacity:0.5;}.cls-1{fill:url(#linear-gradient);}.cls-2{fill:url(#linear-gradient-2);}.cls-3{fill:#f49c14;}.cls-4{fill:url(#linear-gradient-3);}.cls-5{fill:url(#linear-gradient-4);}</style>
```

Comment:
Invalid logo, Can you please correct it.

The company logo has to adher to the following requirements:
Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name
Do not use xlink:href - use inline instead
Do not use title tag
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets
Please refer guidelines here:  https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md.

Reply:
I see, removed the xmlns:xlink